src/components/elementView/powerView/powerView.tsx

```diff
@@ -141,6 +142,26 @@ export class PowerView extends React.PureComponent<
     this.props.updated();
   }
 
+  private isAssetPowerNetworkControlled() {
```

Comment:
wrote this helper method to hopefully help fix the bug but not calling it anywhere